test/integration/kubernetes_deploy_test.rb

```diff
@@ -567,6 +569,25 @@ def test_success_detection_tolerates_out_of_band_deployment_scaling
     assert_equal true, success, "Failed to deploy deployment with dynamic replica count"
   end
 
+  def test_bad_container_on_daemon_sets_fails
+    success = deploy_fixtures("hello-cloud", subset: ["daemon_set.yml"]) do |fixtures|
+      daemon_set = fixtures['daemon_set.yml']['DaemonSet'].first
+      container = daemon_set['spec']['template']['spec']['containers'].first
+      container["image"] = "busybox"
+      container["command"] = ["ls", "/not-a-dir"]
+    end
+
+    refute success
+
+    assert_logs_match_all([
+      "DaemonSet/nginx: FAILED",
+      "The following containers are in a state that is unlikely to be recoverable:",
+      "Events: None found. Please check your usual logging service (e.g. Splunk).",
```

Comment:
My local tests for some reason is not printing any logs or events so I wanted to see if buildkite would do it